exporter/prometheus/prometheus_test.go

```diff
@@ -342,3 +342,84 @@ func TestCumulativenessFromHistograms(t *testing.T) {
 		t.Fatalf("\ngot:\n%s\n\nwant:\n%s\n", got, want)
 	}
 }
+
+func TestConstLabelsIncluded(t *testing.T) {
+	key, label := "service", "spanner"
+	exporter, err := NewExporter(Options{
+		ConstLabels: prometheus.Labels{
+			key: label,
```

Comment:
How is it possible that the literal's name is `key` i.e. unexported, instead of `Key`?
Did you run this test?

Reply:
Yes, I have run the test. `prometheus.Labels` is a map type.

Reply:
Oh, aha gotcha. Let's make that obvious then, I think static assignments take away such ambiguities and also below I added a suggestion to enumerate the output in a statically created string, that way we don't have to dynamically create the output